app/views/shared/_user_status.html.erb

```diff
@@ -0,0 +1,19 @@
+<% if current_user.friend?(user) %>
```

Comment:
- [ ] There shoud be absolutely NO logic in views.
Rails has built-in mechanism to deal with view-related logic which is View Helpers (app/helpers/*_helper.rb). So the common convention is to move any logic in the views to the helpers. Please take a look at [this](https://mixandgo.com/learn/the-beginners-guide-to-rails-helpers) for more information. Let's move all this logic in a `users_helper.rb`